helm/easeprobe/Chart.yaml

```diff
@@ -3,7 +3,7 @@ annotations:
 apiVersion: v2
 appVersion: "2.0.0"
 name: easeprobe
-version: "0.0.1"
+version: "1.0.0"
```

Comment:
Consider adding more metadata to the Chart.yaml file, such as 'keywords', 'icon', and 'type'. This can improve the discoverability and understanding of your Helm chart. [medium]